src/test/e2e/api/admin/event.e2e.test.ts

```diff
@@ -150,3 +151,61 @@ test('can search for events', async () => {
             expect(res.body.events[0].data.id).toEqual(events[1].data.id);
         });
 });
+
+test('event creators - if system user, return system name, else should return name from database if user exists, else from events table', async () => {
```

Comment:
Nice! I'd personally like to see tests for:
- that we have a default for admin token user's name (-42). 
- what happens if multiple events have the same created by user id but different created by names (do we take the first, last, or a random one?)

But I'm happy for that to happen in a follow-up PR. I think the second one is especially interesting, but it's also definitely an edge case. I'll leave that up to you.